From 553c9f59f650c98d7782065b39725c9d2156c233 Mon Sep 17 00:00:00 2001
From: DogeyStamp <adeng.sean@gmail.com>
Date: Tue, 6 Apr 2021 18:28:50 -0400
Subject: [PATCH] Remove tag numbers and center tag indicator

This patch makes tags square, removes the tag numbers, and centers the
indicator showing a window is present so that the bar looks sleeker.
---
 dwm.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/dwm.c b/dwm.c
index b0b3466..88aa701 100644
--- a/dwm.c
+++ b/dwm.c
@@ -433,7 +433,7 @@ buttonpress(XEvent *e)
 	if (ev->window == selmon->barwin) {
 		i = x = 0;
 		do
-			x += TEXTW(tags[i]);
+			x += bh;
 		while (ev->x >= x && ++i < LENGTH(tags));
 		if (i < LENGTH(tags)) {
 			click = ClkTagBar;
@@ -716,11 +716,11 @@ drawbar(Monitor *m)
 	}
 	x = 0;
 	for (i = 0; i < LENGTH(tags); i++) {
-		w = TEXTW(tags[i]);
+		w = bh;
 		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeSel : SchemeNorm]);
-		drw_text(drw, x, 0, w, bh, lrpad / 2, tags[i], urg & 1 << i);
+		drw_text(drw, x, 0, bh, bh, 0, "", urg & 1 << i);
 		if (occ & 1 << i)
-			drw_rect(drw, x + boxs, boxs, boxw, boxw,
+			drw_rect(drw, x+boxw,boxw, w-boxw*2, w-boxw*2,
 				m == selmon && selmon->sel && selmon->sel->tags & 1 << i,
 				urg & 1 << i);
 		x += w;
-- 
2.31.1

